File: src/UrnaEletronica/backend/UrnaEletronica.Dominio/Modelos/Partidos/Partido.cs
using UrnaEletronica.Dominio.Modelos.Candidatos;
using UrnaEletronica.Dominio.Modelos.Coligacoes;

namespace UrnaEletronica.Dominio.Modelos.Partidos
{
    public class Partido
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sigla { get; set; }
        public int ColigacaoId {  get; set; }
        public Coligacao Coligacao { get; set; }
        public IEnumerable<Candidato> Candidatos { get; set; }
    }
}

File: src/UrnaEletronica/frontend/UrnaEletronica-app/src/app/shared/models/interfaces/partido/Partido.ts
import { Candidato } from "../candidato";
import { Coligacao } from "../coligacao";

export interface Partido {
    id: number;
    nome: string;
    sigla: string;
    coligacaoId: number;
    coligacao: Coligacao;
    candidatos: Candidato[];
}
